.travis.yml

```diff
@@ -46,6 +46,7 @@ matrix:
 notifications:
   email: false
 before_script:
+  - echo ./ci/install-julia.sh $JULIA_VERSION
```

Comment:
if you end up needing to make changes to this script I'd still rather "upstream" them and have just one copy that lives in the website repo, once you get it all working